### PR TITLE
tools: add default value of THE_NEW_DDIO_IMPL (fix)

### PR DESCRIPTION
**Pull request by ldorau**
Add default value of THE_NEW_DDIO_IMPL
to prevent the following error:
```
File "rpma/tools/perf/lib/Requirement.py", line 147, in is_met
    if config['THE_NEW_DDIO_IMPL']:
KeyError: 'THE_NEW_DDIO_IMPL'
```

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)

### CODE REVIEW

**Review by janekmi — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)* status: all files reviewed, 1 unresolved discussion (waiting on @ldorau)

___
*[tools/perf/lib/Requirement.py, line 147 at r1](https://reviewable.io/reviews/pmem/rpma/1337#-Mp6zh22C3V46UZ6YKGW:-Mp6zh22C3V46UZ6YKGX:b-w2yohc) ([raw file](https://github.com/pmem/rpma/blob/fd34e554c00bd4e03872e6b7794803dff4e19b56/tools/perf/lib/Requirement.py#L147)):*
> ```Python
>                 # so a dedicated variable won't be needed when the Bash runner will
>                 # be decommissioned.
>                 if config.get('THE_NEW_DDIO_IMPL', True):
> ```

Setting `THE_NEW_DDIO_IMPL` to True will make setting the DDIO in the worst-case twice when using Bash runners without setting `THE_NEW_DDIO_IMPL`. It is fine.

But, maybe it is not to have it set overall is not such a bad idea. No additional variable will be involved. When Python-based runners will be up and running it will be the only place where DDIO is set.

TLDR: Maybe we can remove this variable overall. What do you think?

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)* status: all files reviewed, 1 unresolved discussion (waiting on @janekmi)

___
*[tools/perf/lib/Requirement.py, line 147 at r1](https://reviewable.io/reviews/pmem/rpma/1337#-Mp6zh22C3V46UZ6YKGW:-Mp74FFH0xtm7g1m0zuy:b-wqr3r4) ([raw file](https://github.com/pmem/rpma/blob/fd34e554c00bd4e03872e6b7794803dff4e19b56/tools/perf/lib/Requirement.py#L147)):*
<details><summary><i>Previously, janekmi (Jan Michalski) wrote…</i></summary><blockquote>

Setting `THE_NEW_DDIO_IMPL` to True will make setting the DDIO in the worst-case twice when using Bash runners without setting `THE_NEW_DDIO_IMPL`. It is fine.

But, maybe it is not to have it set overall is not such a bad idea. No additional variable will be involved. When Python-based runners will be up and running it will be the only place where DDIO is set.

TLDR: Maybe we can remove this variable overall. What do you think?
</blockquote></details>

So, do you want to call `cls.__set_DDIO(req, config)` always here?

**Review by janekmi — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)* status: all files reviewed, 1 unresolved discussion (waiting on @ldorau)

___
*[tools/perf/lib/Requirement.py, line 147 at r1](https://reviewable.io/reviews/pmem/rpma/1337#-Mp6zh22C3V46UZ6YKGW:-MpCC872A4f6sjshrDmS:b-u64hzb) ([raw file](https://github.com/pmem/rpma/blob/fd34e554c00bd4e03872e6b7794803dff4e19b56/tools/perf/lib/Requirement.py#L147)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

So, do you want to call `cls.__set_DDIO(req, config)` always here?
</blockquote></details>

Yes. This is my point. It may be a simpler solution than introducing a temporary but very critical configuration variable that is already planned to be decommissioned in the near future.

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)* status: all files reviewed, 1 unresolved discussion (waiting on @janekmi)

___
*[tools/perf/lib/Requirement.py, line 147 at r1](https://reviewable.io/reviews/pmem/rpma/1337#-Mp6zh22C3V46UZ6YKGW:-MpCCpjg50XrfA0n_bUv:b-pa64k6) ([raw file](https://github.com/pmem/rpma/blob/fd34e554c00bd4e03872e6b7794803dff4e19b56/tools/perf/lib/Requirement.py#L147)):*
<details><summary><i>Previously, janekmi (Jan Michalski) wrote…</i></summary><blockquote>

Yes. This is my point. It may be a simpler solution than introducing a temporary but very critical configuration variable that is already planned to be decommissioned in the near future.
</blockquote></details>

Yes, it is a good idea.

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1337)* status: all files reviewed, 1 unresolved discussion (waiting on @janekmi)

___
*[tools/perf/lib/Requirement.py, line 147 at r1](https://reviewable.io/reviews/pmem/rpma/1337#-Mp6zh22C3V46UZ6YKGW:-MpCCyQiCYtEIr-h44kD:b-2kt4n3) ([raw file](https://github.com/pmem/rpma/blob/fd34e554c00bd4e03872e6b7794803dff4e19b56/tools/perf/lib/Requirement.py#L147)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

Yes, it is a good idea.
</blockquote></details>

So I will remove this `if`

**Comment by ldorau** [comment]
Reshaped to: https://github.com/pmem/rpma/pull/1343
